runtime/shiden/src/lib.rs

```diff
@@ -940,6 +940,201 @@ impl InstanceFilter<RuntimeCall> for ProxyType {
         }
     }
 }
+#[cfg(test)]
```

Comment:
Great! :pray: 

Can you please move all the tests at the bottom?
That way we can keep adding on new ones for other pallets too.